suite-common/sentry/src/index.ts

```diff
@@ -67,8 +67,9 @@ const beforeBreadcrumb: Options['beforeBreadcrumb'] = breadcrumb => {
         breadcrumb.data?.url?.contains('data.trezor.io/suite/log');
     const isImageFetch =
         breadcrumb.category === 'xhr' && breadcrumb.data?.url?.contains('/assets/');
+    const isConsole = breadcrumb.category === 'console';
 
-    if (isAnalytics || isImageFetch) {
+    if (isAnalytics || isImageFetch || isConsole) {
```

Comment:
By this we get rid of all `console.logs`. Maybe some `console.logs` can help us debug a bug. Does it make sense to make some exceptions? 

For example, in application log, we track only some redux actions.